README.md

```diff
@@ -59,16 +73,31 @@ If you have multiple version of Python installed, make sure you are using the co
                             The number of quadrant segements (max 8), see:
                             http://postgis.net/docs/ST_Buffer.html. (default:
                             None)
+
+    required named arguments:
+      --travelTimes TRAVELTIMES [TRAVELTIMES ...]
+                            The travel time in seconds as a list of integers.
+                            (default: None)
+      --serviceUrl SERVICEURL
+                            The URL of the Route360° API endpoint. (default: None)
+      --serviceKey SERVICEKEY
+                            Your personal key for the API. (default: None)
+      --travelType TRAVELTYPE
+                            The travel type for the request: car, walk, bike or
+                            transit (default: None)
+      --source SOURCE       The source as doubles (lat,lng) separated by ';'.
+                            (default: None)
       --outputDir OUTPUTDIR
                             The path where to write the output files (default:
                             None)
       --outputFilename OUTPUTFILENAME
-                            The the name of the file to write to (default: None)  
+                            The the name of the file to write to (default: None)
+
 
 ### Example
 
-    python getPolygons.py --travelType walk \
-                   --travelTime 1800 \
+    python -m r360_py.cli --travelType walk \
+                   --travelTimes 1800 \
```

Comment:
Question on this actually... I don't see `travelTime` anywhere in the code, only `travelTimes`. But it works with `travelTime` as well. Why? `travelTimes` functions with a single time, so this should be the way to use it, imho.